### PR TITLE
[CI] test coq 8.20

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -68,7 +68,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         ocaml-compiler: [4.13.x]
-        coq: [8.18.0, 8.19.0, dev]
+        coq: [8.18.0, 8.19.0, 8.20.+rc1, dev]
```
Comment (gares) [inline .github/workflows/ci.yml:71]:
```suggestion
        coq: [8.18.0, 8.19.0, 8.20+rc1, dev]
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -68,7 +68,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         ocaml-compiler: [4.14.x]
-        coq: [8.18.0, 8.19.0, dev]
+        coq: [8.18.0, 8.19.0, 8.20.+rc1, dev]
```
Comment (gares) [inline .github/workflows/ci.yml:71]:
```suggestion
        coq: [8.18.0, 8.19.0, 8.20+rc1, dev]
```

**Comment by gares** [comment]
the problem is another ifdex that I probably fucked up

**Comment by gares** [comment]
`Evd.evar_universe_context` is now called `Evd.ustate`, I forgot to do the PR for master.

**Comment by gares** [comment]
I'm a bit lost, why does nix not pick the same "coq master"?

**Comment by rtetley** [comment]
You have to update it "a la mano" (by changing the pin in the flake.nix) which I just did by rebasing

**Pull request merged**